offlineimap.conf

```diff
@@ -1066,7 +1066,11 @@ remoteuser = username
 #   - header (the Date header of the email)
 #   - imap (let the IMAP server set the internaldate)
 #
-# Default is timestamp
+# If nothing is set, or the given option is failing for some reasens, the following 
```

Comment:
reasons

Reply:
I thought that was the way offlineimap behaved before?

Reply:
It's about the syntax. What's written is "reasens".